content/docs/user-guide/gems/repositories/overview.md

```diff
@@ -0,0 +1,191 @@
+    "repo_uri": "http://localhost:8080/", 
+    "last_updated": "2021-11-18",
+    "sha256":"",
+    "type": "Tool",
+    "summary": "The Downloadable Test Gem provides an example of a downloadable test Gem.",
+    "canonical_tags": [
+        "Gem"
+    ],
+    "user_tags": [
+        "Gameplay",
+        "Simulation",
+        "Utility"
+    ],
+    "icon_path": "preview.png",
+    "requirements": "",
+    "documentation_url": "https://o3de.org/",
+    "dependencies": []
+}
+```
+| Field | Description |
+| --- | --- |
+| origin_uri | The URI for the `.zip` file containing the Gem archive - this is a direct link to download the Gem `.zip` archive. |
+| repo_uri | The URI for the Gem Repository |
+| last_updated | The date this file or Gem was last updated in YYYY-MM-DD format |
+| sha256 | The SHA256 digest of the `.zip` archive pointed to by the `origin_uri` field - for testing you can omit this field, but it is highly recommended you include it. |
+
+## Creating a Gem Repository  
+
+### Using a Template
+
+1. Create a new gem repo using `o3de.bat`.  This example uses a localhost URL as the repository URI for testing.
+
+    ```
+    scripts\o3de.bat create-from-template -tp Templates\GemRepo -dp <repo path on disk> -r ${RepoURI} http://localhost:8080/
+    ```
+
+1. Open the `repo.json` file in a text editor and update all the fields except the gem's list - we'll update that when adding a gem to the repository in the next step.
+    ```
+    {
+        "repo_name":"RepositoryName",
+        "origin":"RemoteOriginName",
+        "repo_uri": "http://localhost:8080/",
+        "summary": "Example Repository",
+        "additional_info": "Example repository hosted on local http server.",
+        "last_updated": "10/26/2021 04:56pm",
+        "gems": [
+        ]
+    }
+    ```
+
+### Add a Gem to the Gem Repository
+1.  Make a folder inside your Gem Repository for the Gem.  In this example we'll assume the gem is named "RemoteExample" and make a folder named "RemoteExampleV1" and place a `gem.json` in there that looks like this:
+    ```
+    {
+        "gem_name": "RemoteExample",
+        "display_name": "Example Remote Gem",
+        "license": "Apache-2.0 Or MIT",
+        "origin":"RemoteOriginName",
+        "origin_uri": "http://localhost:8080/RemoteExampleV1/RemoteExampleV1.zip",
+        "sha256":"",
+        "type": "Code",
+        "summary": "An example remote gem.",
+        "canonical_tags": [
+            "Gem"
+        ],
+        "user_tags": [
+            "Gameplay"
+        ],
+        "icon_path": "preview.png",
+        "requirements": "",
+        "documentation_url": "https://o3de.org/docs/user-guide/gems/reference/gameplay/achievements/",
+        "dependencies": []
+    }
+    ```
+
+    The `origin_uri` points to the location of the gem `.zip` archive that will be downloaded
+
+1. Create a Gem, or take an existing Gem, zip up the contents and place it in the "RemoteExampleV1" folder, and name it `RemoteExampleV1.zip`
```

Comment:
Ah, I'm not sure I understand what you mean - what is "it"?  The Gem?  So, something like, "When zipped the Gem is at the level where the gem.json is at the root of the zip file"?  I'm not sure I understand the last part - do you mean "When zipped, the Gem .zip archive is in the folder alongside the gem.json file"?

Reply:
It's just a small error I've seen a couple of times when giving similar instructions, where the folder was zipped so that that the root of the zip file was the gem folder itself and not the contents (for example gem.zip/ExampleGem/gem.json instead of gem.zip/gem.json). Though this is something we should also attempt to detect this in scripts.